### ISSUE TITLE
[ ERROR ] Part of the nodes was not converted to IR. Stopped.

### ISSUE THREAD

**Issue by Sunhil** [TNTWEN/OpenVINO-YOLOV4#3]
environment: 
unbuntu 16.04
openvino 2020.2
tensorflow 1.12.0

use OpenVINO Model Optimizer to convert .pb(Tensorflow) ->IR(OpenVINO .xml .bin .mapping )
python "/DATA/intel/openvino_2020.2.120/deployment_tools/model_optimizer/mo.py" --input_model frozen_darknet_yolov4_model.pb --transformations_config yolov4.json --batch 1   
[ ERROR ]  List of operations that cannot be converted to Inference Engine IR:
[ ERROR ]      Softplus (72)
[ ERROR ]          detector/cspdarknet-53/Conv/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_1/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_3/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_4/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_5/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_6/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_2/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_7/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_8/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_10/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_11/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_12/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_13/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_14/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_15/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_9/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_16/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_17/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_19/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_20/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_21/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_22/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_23/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_24/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_25/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_26/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_27/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_28/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_29/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_30/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_31/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_32/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_33/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_34/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_35/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_36/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_18/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_37/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_38/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_40/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_41/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_42/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_43/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_44/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_45/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_46/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_47/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_48/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_49/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_50/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_51/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_52/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_53/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_54/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_55/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_56/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_57/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_39/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_58/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_59/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_61/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_62/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_63/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_64/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_65/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_66/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_67/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_68/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_69/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_70/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_60/Softplus
[ ERROR ]          detector/cspdarknet-53/Conv_71/Softplus
[ ERROR ]  Part of the nodes was not converted to IR. Stopped.

Is the vision of OPENVINO cause the questions? And could you tell me the speed of yolov4 in OPENVINO? Thanks a lot！

**Comment by TNTWEN** [comment]
Hi @Sunhil 
Please install OpenVINO 2020R4 which supports softplus

**Comment by TNTWEN** [comment]
Hi @Sunhil
yolov4 ,intel i5 8250U ,FP32,CPU almost1fps     FP32,GPU,almost 2~3FPS   ,FP16 will be faster
You can download the latest version of openvino to do more testing.
I have not tested the specific performance of various equipment at present because I will open a new repository in the future to share OpenVINO YOLOV4/3 model optimization to make it faster.So in this project, I pay more attention to the accuracy of the model compared with darknet and other frameworks.

**Comment by Aavesh** [comment]
I Had the same problem, however as you said got resolved after upgrading openvino. Thanks for your effort with this module.

I am testing Yolov4 model in intel NCS and its very slow. Any optimization tips

**Comment by TNTWEN** [comment]
Hello!@Aavesh 
What's your OpenVINO version now.    Remember to upgarde to OpenVINO 2020R4!!! Only this version support mish  now.

**Comment by TNTWEN** [comment]
@Aavesh 
There are two ways to optimize Yolo: cut layers and cut channels. The former can make the model run faster, and the latter can make the model weight file smaller.
But the optimization model needs a long training time and you need enough GPU resources
I can recommend a project to you:https://github.com/tanluren/yolov3-channel-and-layer-pruning.The contents are all in Chinese and there will be many problems while training.The optimization process can be time-consuming and troublesome.
If you want to convert optimized yolo model to OpenVINO,You also need to be very familiar with the model transformation （darknet ->tf1.x)code.
At the end of September, I will open a new repository to share my optimization process, and I am still trying to reduce the complexity of model optimization.

Thanks for your trying!

**Comment by Aavesh** [comment]
@TNTWEN 
yes i did upgrade to OpenVINO 2020R4 and that's what helped. Thanks for your tip on optimization, will definitely give a shot.

Also thanks again for your work.Best Wishes .
